processing/src/main/java/io/druid/query/Query.java

```diff
@@ -51,49 +52,46 @@
 })
 public interface Query<T>
 {
-  public static final String TIMESERIES = "timeseries";
-  public static final String SEARCH = "search";
-  public static final String TIME_BOUNDARY = "timeBoundary";
-  public static final String GROUP_BY = "groupBy";
-  public static final String SEGMENT_METADATA = "segmentMetadata";
-  public static final String SELECT = "select";
-  public static final String TOPN = "topN";
-  public static final String DATASOURCE_METADATA = "dataSourceMetadata";
+  String TIMESERIES = "timeseries";
+  String SEARCH = "search";
+  String TIME_BOUNDARY = "timeBoundary";
+  String GROUP_BY = "groupBy";
+  String SEGMENT_METADATA = "segmentMetadata";
+  String SELECT = "select";
+  String TOPN = "topN";
+  String DATASOURCE_METADATA = "dataSourceMetadata";
```

Comment:
@cheddar removed that. And fields in public interface are all public static fields even without declaration.